explorer/explorertypes.go

```diff
@@ -51,14 +51,17 @@ type AddressTx struct {
 	FormattedTime string
 	ReceivedTotal float64
 	SentTotal     float64
+	IsFunding     bool
+	MatchedTx     uint64
```

Comment:
This seems to be the source of several of the issues I raised.  For locating an input's/output's corresponding output/input, the row ID in the address table would indeed be needed.  I think this is what you are storing in `MatchedTx` now.  However, for just linking to the transaction page for the transaction of the corresponding in/out, we just need the tx_hash or row ID _in the transactions_ table.

Reply:
To fix this issue we might need to change the addresses table schema so that instead of storing the row id of the matching tx we can store the tx_hash for that specific input/output.